gradle/test.gradle

```diff
@@ -39,19 +39,38 @@ task runTestsWithTag(type: Test) {
                     "${project.properties['dockerRepoPath']}/${project.properties['dockerImageName']}/${project.version}:${project.properties['dockerImageTag']}")
             // Normally CLI version comes from jar. For unit tests, there is no jar, so pass version this way.
             systemProperty("TERRA_CLI_VERSION", "${project.version}")
+
+            if (cloudPlatform == "gcp") {
+                allTestTags.add("unit-gcp")
+            } else if (cloudPlatform == null) {  // run all tests
+                allTestTags.addAll(["unit-gcp"])
```

Comment:
the if else clause are doing the same thing. do you want to add unit-aws or something here?

Reply:
this is intended to support unit-aws in the next PR, and potentially unit-azure later in the year. 
to avoid confusion adding those else clauses here